tools/projmgr/templates/cdefault.yml

```diff
@@ -1,7 +1,5 @@
 default:
 
-  compiler: GCC
```

Comment:
This may break existing projects not setting a default compiler elsewhere.

Reply:
Agreed, I think there is a clear error message indicating what is missing. 
`error csolution: compiler undefined, use '--toolchain' option or add 'compiler: <value>' to yml input`